docs/reference/elaborator-reflection.rst

```diff
@@ -32,6 +32,9 @@ Running Elaborator Scripts
 
 On their own, tactics have no effect.
 The meta-operation ``%runElab script`` runs ``script`` in the current elaboration context.
+Before you can use ``%runElab``, you will have to enable the language extension by adding
+``%language ElabReflection`` in your file (or by passing ``-X ElabReflection`` to the
```

Comment:
If I am not mistaken, should it not be that language extensions need to be enabled both with a pragma and command line option.